src/translations.php

```diff
@@ -136,4 +136,10 @@
         "Longest Streak" => "最长连续贡献",
         "Present" => "至今",
     ],
+     "ta" => [
+        "Total Contributions" => "மொத்த பங்களிப்புகள்",
+        "Current Streak" => "மிக சமீபத்திய தொடர்ச்சியான பங்களிப்புகள்",
```

Comment:
I've implemented the fix to allow multiple lines. Currently it looks like this:

![image](https://user-images.githubusercontent.com/20955511/196019322-97dd5488-617b-4015-8733-7557bd2200eb.png)

Using your second translation it fits a bit better:

![image](https://user-images.githubusercontent.com/20955511/196019353-4dba2f0c-f4f1-42af-8a8a-6e4e6d62ad56.png)

If you agree that it would be better with the shorter version, let me know, or you can click Commit Suggestion below to push it to the branch.

```suggestion
        "Current Streak" => "மிக சமீபத்திய பங்களிப்புகள்",
```